andela/apollo-ah-backend
# #164198173 Setup eslint

#### What does this PR do?
Set up eslint for linting
#### Description of Task to be completed?
- create an eslintrc file
- configure eslint with airbnb style guide
- Install development dependencies needed for eslint set up
#### How should this be manually tested?
- Follow the steps below:
  - Run the following commands in your terminal
`git clone https://github.com/andela/apollo-ah-backend.git`
`cd apollo-ah-backend`
- Run `npm install` to install all dependencies.
- Create a javascript file
- Paste the following code in the javascript file created
`let name = "Javascript";`
- You should get an eslint warning
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#64198173](https://www.pivotaltracker.com/story/show/64198173)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
